custom_components/flair/cover.py

```diff
@@ -0,0 +1,133 @@
+import logging
+from homeassistant.util.percentage import ordered_list_item_to_percentage
+from homeassistant.exceptions import PlatformNotReady
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.components.cover import (
+    CoverEntity,
+    SUPPORT_SET_TILT_POSITION,
+    DEVICE_CLASS_DAMPER,
```

Comment:
DEVICE_CLASS* are deprecated as of 2021.12. Need to use `CoverDeviceClass` enum instead.